ios/Runner/AppDelegate.swift

```diff
@@ -53,16 +53,16 @@ import os.log
 
         // Stub, this is used with Android to deliver authorization results
         // after the app died in the background while the user was using the browser
-        // to authenticate. This, the app dying in the backgroudn, does not happen on iOS
-        let appAuthEvents = FlutterEventChannel.init(name: APP_AUTH_EVENTS,
-                                                     binaryMessenger: controller.binaryMessenger)
+        // to authenticate. This, the app dying in the background, does not happen on iOS
+        let appAuthEvents = FlutterEventChannel(name: APP_AUTH_EVENTS,
+                                                binaryMessenger: controller.binaryMessenger)
         appAuthEvents.setStreamHandler(StubEventStreamHandler())
-
+        
```

Comment:
This kind of whitespace is super weird to me, but ok :D

Reply:
🤭ups.  The .init and a typo was the main reason for the cosmetics.